source/dem/particle_particle_fine_search.cc

```diff
@@ -86,7 +80,8 @@ ParticleParticleFineSearch<dim>::particle_particle_fine_search(
                 {
                   // Getting the particle one contact list and particle two id
                   auto particle_one_contact_list =
-                    &local_adjacent_particles[particle_one_id];
+                    &container_manager
+                       .local_adjacent_particles[particle_one_id];
```

Comment:
same here for the local adjacent particles. Pre store the referene to it in another variable.